src/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue

```diff
@@ -0,0 +1,341 @@
+<template>
+  <div id="summary-business-home-jurisdiction">
+    <section :class="{ 'invalid-section': !isContinuationInStep1Valid }">
+      <div
+        v-if="!isContinuationInStep1Valid"
+        class="businessHomeStepErrorMessage"
+      >
+        <span>
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text mx-1">This step is unfinished.</span>
+
+          <router-link
+            :to="{ path: `/${RouteNames.CONTINUATION_IN_BUSINESS_HOME}` }"
+          >Return to this step to finish it</router-link>
+        </span>
+      </div>
+
+      <div class="existing-business-information-summary">
+        <!-- Home Jurisdiction -->
+        <article class="section-container">
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              sm="3"
+              class="pr-4"
+            >
+              <label>Home Jurisdiction</label>
+            </v-col>
+            <v-col
+              cols="12"
+              sm="9"
+              class="pt-4 pt-sm-0"
+            >
+              <div id="home-jurisdiction">
+                {{ homeJurisdiction || '[Unknown]' }}
```

Comment:
Note that the fallback values are in the template here. I think this is a good general practice as it allows the template to do logic if the value is null.